src/NuGetGallery/Controllers/PackagesController.cs

```diff
@@ -2967,5 +2996,13 @@ private static HttpStatusCodeResult HttpForbidden()
         {
             return new HttpStatusCodeResult(HttpStatusCode.Forbidden, Strings.Unauthorized);
         }
+
+        private static bool isSupportedSortBy(string sortBy)
```

Comment:
Nit: C# style is to uppercase all the things

```suggestion
        private static bool IsSupportedSortBy(string sortBy)
```